src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -19,6 +19,76 @@
 
 
 # -------------------------------------------- ops ------------------------------------------------
+@binary_op
+def bitwise_and(
+    left_node: NodeInput,
+    right_node: NodeInput,
+    auto_broadcast: str = "NUMPY",
+    name: Optional[str] = None,
+) -> Node:
+    """Return node which performs bitwise AND operation on input nodes element-wise.
+
+    For boolean input tensors, operator is equivalent to logical_and.
+
+    :param left_node: Tensor of integer or boolean datatype providing data.
+    :param right_node: Tensor of integer or boolean datatype providing data.
+    :param auto_broadcast: The type of broadcasting specifies rules used for auto-broadcasting of input tensors. Defaults to “NUMPY”.
```

Comment:
Just a side note. What are others types of broadcasting? It might be worth to mention them in all examples.

Reply:
https://docs.openvino.ai/nightly/openvino_docs_ops_broadcast_rules.html
Based on docs, choices for elementwise ops like add/bitwise are: None, PDPD, NUMPY